.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules/
+models/mongodb/logs.mongod.log
+models/mongodb/data/db/
+models/mongodb/logs/
```

Comment:
This location isn't right, `logs` shouldn't exist under `models`. Change the config to logs are outside of this dirctory.